gptel-context.el

```diff
@@ -444,7 +444,9 @@ If non-nil, indicates backward movement.")
   "Mark gptel context chunk at point for removal."
   (interactive)
   (let* ((overlays (if (use-region-p)
-                       (overlays-in (region-beginning) (region-end))
+                       (cl-remove-if-not (lambda (ov)
```

Comment:
What do you mean by "region selection overlay"?

Reply:
The actual overlay used by Emacs when you highlight a region (i.e. the one created by `activate-mark`).

Reply:
Ah okay.  I think it's cleaner to just add a `(when (overlay-get ov 'gptel-overlay) ...)` inside the `dolist` a few lines below.

Reply:
I'll change it to that then.